docs/installation.md

```diff
@@ -34,12 +34,12 @@
 - Linux: `apt-get install python-protobuf`
 
 ### Get encrypt.so (Windows part writing need fine tune)
-We don't have the copyright of encrypt.so, please grab from internet and build your self.Take the risk as your own.
-Example build sequence:
-Create a new separate folder some here
+Due to copywrite on the encrypt.so we are not directly hosting it. Please find a copy elsewhere on the internet and compile it yourself. We accept no responsibility should you encounter any problems with files you download elsewhere.
+
+Ensure you are in the PokemonGo-Bot main folder and run:
+
+`wget http://pgoapi.com/pgoencrypt.tar.gz && tar -xf pgoencrypt.tar.gz && cd pgoencrypt/src/ && make && mv libencrypt.so ../../encrypt.so`
```

Comment:
@nivong by second I mean 'alternative', or even just a line to tell OSX users to substitute curl in for wget. To be honest it's not clear to me exactly if we're even going to keep this part of the docu since it's handled in setup.sh(?) now - but while it's there I figure it should be maintained.

Depends what you think?